scripts/configure.sh

```diff
@@ -41,7 +41,7 @@ fi
 
 if [[ "$OSTYPE" == "linux-gnu"* ]]; then
   apt update
-  apt install -y build-essential wget cmake libgtest-dev libbenchmark-dev lcov git software-properties-common rsync unzip
+  apt install -y build-essential wget cmake libgtest-dev libbenchmark-dev lcov git software-properties-common rsync unzip python3-dev
```

Comment:
Verified that this works. It installed python 3.10 in my Ubuntu environment, but in `Dockerfile` at root dir, it is specified that we use python 3.8 `ENV PY_VER=python3.8` (line 30)